forc-plugins/forc-node/src/consts.rs

```diff
@@ -0,0 +1,34 @@
+/// Minimum fuel-core version supported.
+pub const MIN_FUEL_CORE_VERSION: &str = "0.40.0";
```

Comment:
Will this ever need to be updated?

Reply:
This can be kept same or updated given the breaking updates to fuel-core run cli interface. Until we have a breaking change there this can stay same, but this gives us a chance to control the environment forc-node is run. Because it is possible that the user is not using fuelup and might be maintaining their own versions